projects/i18n/languages/dutch/core.ts

```diff
@@ -1,5 +1,5 @@
 import {LanguageCore} from '@taiga-ui/i18n/interfaces';
-import {TUI_ENGLISH_LANGUAGE_COUNTRIES} from '@taiga-ui/i18n/languages/english';
+import {TUI_DUTCH_LANGUAGE_COUNTRIES} from '@taiga-ui/i18n/languages/dutch';
```

Comment:
Inside one entry point paths must be relative. They way it is written now is for cross points references. That's why you are getting cycling dependency on itself. This should be relative here.